src/util.js

```diff
@@ -141,7 +168,7 @@ function serialize (node) {
  * Deserialize CBOR block into the internal representation.
  *
  * @param {Uint8Array} data - Binary representation of a CBOR block
- * @returns {Object} - An object that conforms to the IPLD Data Model
+ * @returns {{ [key: string]: any }} - An object that conforms to the IPLD Data Model
```

Comment:
This is probably wrong, probably should just be `{any}`? It could be an object, array, number, string, CID, Uint8Array or null.

Maybe we need a typedef for the data model! That could come in handy for places like this.

Reply:
Good catch, I've changed it to `any`